docs/wiki/index.md

```diff
@@ -1,4 +1,4 @@
-osquery is an operating system instrumentation framework for Windows, OS X (macOS), Linux, and FreeBSD. The tools make low-level operating system analytics and monitoring both performant and intuitive.
+osquery is an operating system instrumentation framework for Windows, OS X (macOS), and Linux. The tools make low-level operating system analytics and monitoring both performant and intuitive.
```

Comment:
There's more also here
https://github.com/osquery/osquery/pull/7508/files#diff-6872e0acfa62253b9d51742be4907abc9d74cb0de73ff33ef4e3346bc2f8c9ddR7

and here
https://github.com/osquery/osquery/pull/7508/files#diff-6872e0acfa62253b9d51742be4907abc9d74cb0de73ff33ef4e3346bc2f8c9ddR11